bitnami/ejbca/templates/deployment.yaml

```diff
@@ -0,0 +1,173 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "ejbca.fullname" . }}
+  labels: {{- include "ejbca.labels" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "ejbca.tplValue" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "ejbca.tplValue" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  selector:
+    matchLabels: {{- include "ejbca.matchLabels" . | nindent 6 }}
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels: {{- include "ejbca.labels" . | nindent 8 }}
+        {{- if .Values.podLabels }}
+        {{- include "ejbca.tplValue" (dict "value" .Values.podLabels "context" $) | nindent 8 }}
+        {{- end }}
+      annotations:
+        checksum/secret: {{ include (print $.Template.BasePath "/secrets.yaml") . | sha256sum }}
+      {{- if .Values.podAnnotations }}
+        {{- include "ejbca.tplValue" (dict "value" .Values.podAnnotations "context" $) | nindent 8 }}
+      {{- end }}
+    spec:
+{{- include "ejbca.imagePullSecrets" . | nindent 6 }}
+      {{- if .Values.podSecurityContext.enabled }}
+      securityContext:
+        runAsUser: {{ .Values.podSecurityContext.runAsUser }}
```

Comment:
AFAIK what we usually do is set fsGroup for the deployment, if podSecurityContext.enabled is set, and then runAsUser for securityContext if containedSecurityContext.enabled is set. It seems there is already a bit of inconsistency related to this point, so I think we should double-check this. Just FYI, I think we were standarizing on the approach set by `bitnami/redis-cluster`.

Reply:
You are right, I have just seen another PR [review](https://github.com/bitnami/charts/pull/2961#discussion_r448145357) where this is also highlighted. I will change that in order to follow the standardization 😁